astropy/modeling/rotations.py

```diff
@@ -187,9 +187,6 @@ def evaluate(cls, x, y, angle):
         single coordinate pair.
         """
 
-        if x.shape != y.shape:
-            raise ValueError("Expected input arrays to have the same shape")
```

Comment:
I'll look into the other issue, but in any case I think this is a workaround to an unrelated problem.